server/decorators/expandedDeliusServiceUserDecorator.test.ts

```diff
@@ -191,81 +191,57 @@ describe(ExpandedDeliusServiceUserDecorator, () => {
             expect(serviceUser.address).toBeNull()
           })
         })
-      })
-
-      describe('null  values checks', () => {
-        describe('when there is no address returned from nDelius', () => {
-          it('returns null', () => {
-            const serviceUser = new ExpandedDeliusServiceUserDecorator(
-              expandedDeliusServiceUserFactory.build({
-                contactDetails: {
-                  addresses: [],
-                },
-              })
-            )
 
-            expect(serviceUser.address).toBeNull()
-          })
-        })
-        describe('when the addresses contains a single undefined address', () => {
+        describe('when all address have a "to" field', () => {
```

Comment:
The (obfuscated) data in question triggering the current `TypeError: Cannot read property 'addressNumber' of undefined` exception is
```
{
  "contactDetails": {
    "addresses": [
      {
        "from": "2021-04-09",
        "to": "2021-04-30",
        "noFixedAbode": true,
        "postcode": "xxx",
        "status": {
          "code": "P",
          "description": "Previous"
        },
        "typeVerified": false,
        "createdDatetime": "2021-04-09T17:07:30",
        "lastUpdatedDatetime": "2021-06-30T10:53:30"
      },
      {
        "from": "2020-06-29",
        "to": "2021-04-09",
        "noFixedAbode": false,
        "addressNumber": "xxx",
        "streetName": "xxx",
        "district": "xxx",
        "postcode": "xxx",
        "status": {
          "code": "P",
          "description": "Previous"
        },
        "typeVerified": false,
        "createdDatetime": "2020-10-09T11:08:00",
        "lastUpdatedDatetime": "2021-04-09T17:04:25"
      }
    ]
  }
}
```
I'd find it useful to describe the real-life scenario as a test case, perhaps "there is no current address added for the service user"?

Reply:
Sure. I've added that into the test. 
The test doesn't have all the fields as you show above because we don't declare all the fields in typescript.